src/uucore/src/lib/features/checksum.rs

```diff
@@ -565,18 +607,18 @@ where
                 // Do the checksum validation
                 if expected_checksum == calculated_checksum {
                     if !quiet && !status {
-                        println!("{prefix}{filename_to_check}: OK");
+                        println!("{prefix}{filename_lossy}: OK");
                     }
                     correct_format += 1;
                 } else {
                     if !status {
-                        println!("{prefix}{filename_to_check}: FAILED");
+                        println!("{prefix}{filename_lossy}: FAILED");
                     }
                     res.failed_cksum += 1;
                 }
             } else {
-                if line.is_empty() {
-                    // Don't show any warning for empty lines
+                if line.is_empty() || line_bytes.starts_with(b"#") {
+                    // Don't show any warning for empty or commented lines
                     continue;
                 }
```

Comment:
This adds support for commented-out lines. Please don't sneakily implement features like this by hiding it inside a bigger changeset. Also, please add tests to check that this actually works as intended, to prevent future regressions.

Reply:
Ok, I am moving this into its own PR

Reply:
Check #6782